src/applications/vaos/components/ExpressCareCard.jsx

```diff
@@ -6,16 +6,16 @@ import { getPatientPhone, getPatientEmail } from '../services/appointment';
 import { APPOINTMENT_STATUS } from '../utils/constants';
 import ExpressCareStatus from './ExpressCareStatus';
 
-export default function ExpressCareListItem({
+export default function ExpressCareCard({
   appointment,
-  index,
   cancelAppointment,
   showCancelButton,
+  headingLevel = '3',
```

Comment:
Heading level of the main title of the card differs between the list and the confirmation page